Fortinet/fortigate/_meta/fields.yml

```
@@ -0,0 +1,29 @@
+rule.apprisk:
+  name: rule.apprisk
+  type: keyword
+  description: ""
+
+dns.rrname:
+  name: dns.rrname
+  type: keyword
+  description: ""
+
+tmp.icmp_code:
+  name: tmp.icmp_code
+  type: keyword
+  description: ""
+
+tmp.icmp_type:
+  name: tmp.icmp_type
+  type: keyword
+  description: ""
```

```suggestion
tmp.icmp_code:
  name: tmp.icmp_code
  type: keyword
  description: ""

tmp.icmp_type:
  name: tmp.icmp_type
  type: keyword
  description: ""
```

Could you please use [ECS ICMP fields](https://www.elastic.co/guide/en/beats/packetbeat/current/exported-fields-icmp.html)